### ISSUE TITLE
Testing - News page

### ISSUE THREAD

**Issue by misspastelwitch** [fdnd-agency/nebulaxplorer#178]
<img width="858" height="803" alt="Image" src="https://github.com/user-attachments/assets/35732235-b869-4918-a5bc-3435f28b1f65" />
**Lighthouse test**

- heading elements are not in a sequentially descending order (titles of news cards are H3 at the moment, but could be h4s)
- Alt text on 'read more' card can be removed as it is the same as the actual text.

More in-depth testing should be done once we have a live-link.

**Comment by misspastelwitch** [comment]
## Accessibility test 2

Once again I am waiting for a live link in order to do more complete testing. 

**Lighthouse test**
<img width="933" height="747" alt="Image" src="https://github.com/user-attachments/assets/9610ff6b-c3c2-4171-9a92-c3b9fb2409e0" />

<img width="932" height="671" alt="Image" src="https://github.com/user-attachments/assets/26cc0529-e17b-4bef-b0b2-297c0d3b2021" />

## Issues:

### Accessibility
- [ ] Background and foreground colours do not have a sufficient contrast ratio, specifically the orange text, which needs to be changed from --cleanroom-140 to --cleanroom-160.
- [x] Document doesn't have a `<title>` element

### Performance

- [ ] Minify JavaScript
- [ ] Reduce unused JavaScript
